.github/workflows/ci.yaml

```diff
@@ -8,9 +8,23 @@ on:
     - cron: "0 4 * * *"
 
 jobs:
-  build-test-documentation:
+  build-test:
     name: Build, test & documentation
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - package: plutus-cbor
+          - package: plutus-merkle-tree
+          - package: hydra-tui
+          - package: hydra-node
+            bench:
```

Comment:
How are these used? I don't see any reference to `bench` anywhere else in this file and looking at the CI execution trace I don't see they are executed.

Reply:
I see they are not referenced as in example https://docs.github.com/en/github-ae@latest/actions/using-jobs/using-a-build-matrix-for-your-jobs#example-running-multiple-versions-of-nodejs

Reply:
This was simply to check that the workflow syntax was okay :blush: ... Going step-by-step, I wanted to introduce jobs taking care of the benchmarks later. 

Thus indeed, for now, they aren't used.